CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+* [CHANGE] Memberlist: allow specifying address and port advertised to the memberlist ring by setting the following configuration:
```

Comment:
Nit: Instead of "memberlist ring", we should say "memberlist cluster members".

(Cortex Ring contains Address field as well, but that isn't affected by these flags.)